azkaban-common/src/main/java/azkaban/utils/YarnUtils.java

```diff
@@ -0,0 +1,71 @@
+package azkaban.utils;
```

Comment:
Please add a header. Check other code files and copy from there. Make sure the year is 2022

Reply:
added